### PR TITLE
User Notify endpoint [SA-14401] (develop)

### PR DESCRIPTION
**Pull request by JustinAClarke**

### CODE REVIEW

File: openapi/components/requestBodies/userNotify.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          example: |
+            Checkout the 22.1 Release notes
+        from:
+          type: object
+          required:
+            - fullName
+          properties:
+            fullName:
+              type: string
+              required: true
```
Comment (jleckie) [inline openapi/components/requestBodies/userNotify.yaml:24]: The comment says this is not required, but then the schema says it is?

File: openapi/components/requestBodies/userNotify.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      type: object
+      required:
+        - message
+        - from
+      properties:
+        message:
+          type: string
+          required: true
+          description: |
+            Message title for the notification
```
Comment (jleckie) [inline openapi/components/requestBodies/userNotify.yaml:14]: Think this should just be described as "The content of the notification message"..  Not sure 'title' is relevant as its not really a title.

File: openapi/openapi.yaml [hunk 3]
```diff
@@ -80,6 +80,11 @@ tags:
     description: |
       Create and manage school news (also known as communications).
 
+  - name: notification
```
Comment (jleckie) [inline openapi/openapi.yaml:83]: We probably don't need a new section, the endpoint exists under the 'user' area so we can probably just keep this as a part of the 'User' section.

File: openapi/components/requestBodies/userNotify.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+              type: string
+              required: true
+              description: |
+                User who initially created this message, if unset defaults to Unknown User
+              example: Schoolbox Admin
+        href:
+          type: string
+          description: |
+            url for when clicking the notification to redirect to, if unset, defaults to #
+          example: https://help.schoolbox.com.au/homepage/3343
```
Comment (jleckie) [inline openapi/components/requestBodies/userNotify.yaml:32]: Should we make point about it being a relative URL or absolute URL.  Do we prefer relative over absolute?

Comment (JustinAClarke) [reply openapi/components/requestBodies/userNotify.yaml:29]: Both will work, this was specifically just an example

**Pull request merged**